mongo/integration/mtest/setup.go

```diff
@@ -186,6 +188,20 @@ func Setup(setupOpts ...*SetupOptions) error {
 		}
 	}
 
+	if testContext.topoKind == LoadBalanced {
```

Comment:
This change is due to a spec/setup requirement - when mongo-orchestration starts up a cluster, it provides a URI that includes authentication information (if needed) but does not include TLS information (e.g. CA file path). We normally work around this by manually modifying the cluster URI, which is done in `getClusterConnString()`, which internally calls `addTLSConfig()`. For LB testing, we have to make the same modification for the URIs of the LBs themselves as well, so we do all of that here and then store the processed URI in `testContext`.